package.json

```diff
@@ -39,8 +39,8 @@
     "dependencies": {
         "@aws-sdk/client-ses": "^3.49.0",
         "@aws-sdk/credential-provider-node": "^3.49.0",
-        "@flowforge/forge-ui-components": "^0.1.2",
-        "@flowforge/localfs": "^0.3.0",
+        "@flowforge/forge-ui-components": "^0.4.0",
```

Comment:
forge-ui-components operates differently to the other `@flowforge` packages, and should not be set at the same version number (yet)

Reply:
```suggestion
        "@flowforge/forge-ui-components": "^0.1.2",
```